src/services/api/utils.ts

```diff
@@ -24,13 +25,22 @@ export class RequireHiveVerify implements CanActivate {
 
 @Injectable()
```

Comment:
Doesn't the JwtService need to be marked as a dependency here?

Reply:
Injectable is just a decorator signalling to nest; this class is open to dependency injection. The module defined jwtService as a provider by importing JwtModule. This class has jwtService dependency injected in the constructor on line 28.

Your comment did make me realise, however, the user details interceptor wasn't being supplied as a provider in its respective client modules so I remedied that in my most recent commit

Reply:
alrighty